python/lambdas/pause-ingest/test_pause_ingest.py

```diff
@@ -0,0 +1,188 @@
+
+    lambda_mock.list_event_source_mappings.side_effect = [
+        {'EventSourceMappings': [{'UUID': 'agg-uuid-1'}]},
+        {'EventSourceMappings': [{'UUID': 'agg-uuid-2'}]},
+        {'EventSourceMappings': [{'UUID': 'court-uuid'}]}
+    ]
+
+    event = {'pause': True}
+    pause_ingest.lambda_handler(event, None)
+
+    eventbridge_mock.put_events.assert_called_once_with(
+        Entries=[{
+            "Source": "ingest-pause",
+            "DetailType": "DR2DevMessage",
+            "Detail": '{"slackMessage": ":alert-noflash-slow: Ingest has been paused in environment test"}',
+            "EventBusName": "default"
+        }]
+    )
+
+    lambda_mock.update_event_source_mapping.assert_has_calls([
+        call(UUID='agg-uuid-1', Enabled=False),
+        call(UUID='agg-uuid-2', Enabled=False),
+        call(UUID='court-uuid', Enabled=False)
+    ], any_order=True)
+
+    args, kwargs = ssm_mock.put_parameter.call_args
+    value = json.loads(kwargs['Value'])
+    assert value['maxConcurrency'] == 0
+    assert value['previousMaxConcurrency'] == 5
+
+def test_resume_ingest(env, boto3_mocks):
+    eventbridge_mock, lambda_mock, ssm_mock = boto3_mocks
+
+    setup_ssm(ssm_mock, max_concurrency=0, previous_max=5)
+
+    lambda_mock.list_event_source_mappings.side_effect = [
+        {'EventSourceMappings': [{'UUID': 'agg-uuid-1'}]},
+        {'EventSourceMappings': [{'UUID': 'agg-uuid-2'}]},
+        {'EventSourceMappings': [{'UUID': 'court-uuid'}]}
+    ]
+
+    event = {'pause': False}
+    pause_ingest.lambda_handler(event, None)
+
+    eventbridge_mock.put_events.assert_called_once_with(
+        Entries=[{
+            "Source": "ingest-pause",
+            "DetailType": "DR2DevMessage",
+            "Detail": '{"slackMessage": ":white_check_mark: Ingest has been resumed in environment test"}',
+            "EventBusName": "default"
+        }]
+    )
+
+    lambda_mock.update_event_source_mapping.assert_has_calls([
+        call(UUID='agg-uuid-1', Enabled=True),
+        call(UUID='agg-uuid-2', Enabled=True),
+        call(UUID='court-uuid', Enabled=True)
+    ], any_order=True)
+
+    args, kwargs = ssm_mock.put_parameter.call_args
+    value = json.loads(kwargs['Value'])
+    assert value['maxConcurrency'] == 5
+    assert 'previousMaxConcurrency' not in value
+
+def test_scheduled_event_both_enabled_max_gt_zero(env, boto3_mocks):
+    eventbridge_mock, lambda_mock, ssm_mock = boto3_mocks
+    lambda_mock.list_event_source_mappings.side_effect = [
+        {'EventSourceMappings': [{'UUID': 'agg-uuid-1', 'Enabled': True}]},
+        {'EventSourceMappings': [{'UUID': 'agg-uuid-2', 'Enabled': True}]},
+        {'EventSourceMappings': [{'UUID': 'court-uuid', 'Enabled': True}]}
+    ]
+    setup_ssm(ssm_mock, max_concurrency=5)
+    event = {'source': 'aws.events'}
+    pause_ingest.lambda_handler(event, None)
+    eventbridge_mock.put_events.assert_not_called()
+
+def test_scheduled_event_agg_disabled(env, boto3_mocks):
```

Comment:
Maybe we could add a more descriptive title e.g.something like "should send message to eventbridge that it's still paused if agg disabled"